warehouse/core/src/main/java/datawave/ingest/util/cache/Loader.java

```diff
@@ -165,6 +165,10 @@ public void run() {
         
     }
     
+    public boolean equals(Loader loader) {
+        return (this.hashCode() == loader.hashCode());
+    }
```

Comment:
why not a more formal equals method, like this?
```suggestion
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Loader<?, ?> loader = (Loader<?, ?>) o;
        return childHash == loader.childHash && lazy == loader.lazy && Objects.equals(entryCache, loader.entryCache) && Objects.equals(executor, loader.executor) && Objects.equals(children, loader.children);
    }
```

Reply:
okay